spec/mailers/producer_mailer_spec.rb

```diff
@@ -47,12 +46,8 @@
     order.save
     order
   end
-  let(:mail) { ActionMailer::Base.deliveries.last }
 
-  before do
-    ActionMailer::Base.deliveries.clear
-    ProducerMailer.order_cycle_report(s1, order_cycle).deliver
-  end
+  let(:mail) { ProducerMailer.order_cycle_report(s1, order_cycle).deliver }
```

Comment:
Do we need `deliver` here? I didn't see that in the documented example.

Reply:
Good point. I'm not sure I tried removing it

Reply:
Certainly not needed. Thanks @mkllnk !